### ISSUE TITLE
Code Insights: Opacity animation leads to the bad performance in Safari

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#40162]
In Code Insights, we use standard animation properties for active/focus series appearance animation. 
```
 transitionProperty: 'opacity',
 transitionDuration: '200ms',
 transitionTimingFunction: 'ease-out',
```

It turned out that even with partial lazy rendering on the all insights dashboard, these animation rules lead to performance problems for all charts.

But Chrome and Firefox don't have this problem even with a big number of DOM elements on the page.

**Comment by felixfbecker** [comment]
In WebKit animations on `opacity` trigger layout, paint and compositing: https://csstriggers.com/

Maybe using `will-change` would help: https://web.dev/animations-guide/#force

**Comment by vovakulikov** [comment]
Good call. Yeah, I think this is what I saw in dev tools happened on the dashboard page. (didn't know that opacity triggers the full flow of "rendering" in WebKit) Thanks
